### PR TITLE
Implement timeseries Dto

### PR DESCRIPTION
**Pull request by petterwildhagen**

### CODE REVIEW

File: backend/api/Dtos/TimeSeriesDto.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    {
+
+    }
+
+    public class TimeMassVolumeDto : TimeSeriesDto<double>
+    {
+
+    }
+
+    public class TimeSeriesSchedule : TimeSeries<int>
```
Comment (ericnorway) [inline backend/api/Dtos/TimeSeriesDto.cs:33]: Rename to TimeSeriesScheduleDto

File: backend/api/Dtos/ExplorationDto.cs [hunk 2]
```diff
@@ -9,13 +9,13 @@ public class ExplorationDto
...
-        public GAndGAdminCostDto? GAndGAdminCost { get; set; }
+        public ExplorationCostProfileDto CostProfile { get; set; } = null!;
+        public ExplorationDrillingScheduleDto DrillingSchedule { get; set; } = null!;
+        public GAndGAdminCostDto GAndGAdminCost { get; set; } = null!;
         public double RigMobDemob { get; set; }
     }
 
-    public class ExplorationCostProfileDto : TimeSeriesCost { }
+    public class ExplorationCostProfileDto : TimeSeriesCostDto { }
     public class ExplorationDrillingScheduleDto : TimeSeriesSchedule { }
```
Comment (ericnorway) [inline backend/api/Dtos/ExplorationDto.cs:19]: TimeSeriesScheduleDto

File: backend/api/Dtos/WellProjectDto.cs [hunk 3]
```diff
@@ -18,7 +18,7 @@ public class WellProjectDto
         public double PluggingAndAbandonment { get; set; }
     }
 
-    public class WellProjectCostProfileDto : TimeSeriesCost
+    public class WellProjectCostProfileDto : TimeSeriesCostDto
     {
     }
     public class DrillingScheduleDto : TimeSeriesSchedule
```
Comment (ericnorway) [inline backend/api/Dtos/WellProjectDto.cs:24]: TimeSeriesScheduleDto

File: backend/api/Dtos/TimeSeriesDto.cs (hunk 1 above)
Comment (petterwildhagen) [reply backend/api/Dtos/TimeSeriesDto.cs:33]: Fixed.

File: backend/api/Dtos/ExplorationDto.cs (hunk 2 above)
Comment (petterwildhagen) [reply backend/api/Dtos/ExplorationDto.cs:19]: Fixed (by updating in TimeSeriesDto class)

File: backend/api/Dtos/WellProjectDto.cs (hunk 3 above)
Comment (petterwildhagen) [reply backend/api/Dtos/WellProjectDto.cs:24]: Fixed (by updating in TimeSeriesDto class)

**Review by ericnorway — APPROVED** [review]
looks good

**Review by ericnorway — APPROVED** [review]
looks good

**Pull request merged**